### PR TITLE
Feature/additional webfiling screens

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/integration/miscellaneous_screens/check_additional_screens.spec.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        manageYourEremindersPage.useWebfilingEmailAddress();
+        cy.accessibilityCheck();
+
+        manageYourEremindersPage.addEmailButton();
+        cy.accessibilityCheck();
+
+        manageYourEremindersPage.viewTermsOfOperation();
+        cy.accessibilityCheck();
+        // Go back to company overview screen to exit the test
+        cy.visit('https://ewf-kermit.companieshouse.gov.uk//profile')
```
Comment (rsuller) [inline cypress/integration/miscellaneous_screens/check_additional_screens.spec.js]: You don't need to pass the entire url here. For this one you can just pass `cy.visit('/profile')` this is important if we should move to other environments.

File: cypress/integration/miscellaneous_screens/check_additional_screens.spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        cy.accessibilityCheck();
+        
+    })
+
+    it('Company Authentication', () => {
+        companyOverview.selectLinkWithText('Company authentication');
+        cy.accessibilityCheck();
+    })
+
+    it.only('Change Account Details', () => {
```
Comment (rsuller) [inline cypress/integration/miscellaneous_screens/check_additional_screens.spec.js]: I am not convinced that this test is doing everything that you expect. When running this in interactive mode it only seems to run this `it` block (change Account Details). I would assume this is due to the `afterEach` method of signing out.
I am not convinced the `afterEach` is giving us anything and maybe we should consider using the `cy.visit` to return to the company overview in the `beforeEach` method?

File: cypress/support/page_objects/ManageYourEremindersPage.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        //wait for fields to be displayed
+        cy.get('#addEmailLink').click().wait(2000);
+    }
+
+    useWebfilingEmailAddress() {
+        cy.get('input[type="checkbox"]').check();
+    }
+
+    addEmailButton() {
+        //cy.get('input[type="submit"][value="Add"]').click();
```
Comment (rsuller) [inline cypress/support/page_objects/ManageYourEremindersPage.js]: Can this line be removed?

File: cypress/support/page_objects/ManageYourEremindersPage.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        cy.get('[name="eRemindersDetails.AddEmailCtrl.btnAdd"]').click();
+    }
+
+    viewTermsOfOperation() {
+        // As this link opens in a new window, call cy.visit() in oder to test in isolation
+        cy.visit('https://ewf-kermit.companieshouse.gov.uk/sframe?name=eRemindersTermsOfOperation');
```
Comment (rsuller) [inline cypress/support/page_objects/ManageYourEremindersPage.js]: Similar to other comments you can replace with `cy.visit('sframe?name=eRemindersTermsOfOperation')`

File: cypress/integration/miscellaneous_screens/check_additional_screens.spec.js (hunk 2 above)
Comment (danfalc) [reply cypress/integration/miscellaneous_screens/check_additional_screens.spec.js]: This shouldn't be `it.only` should be `it`. I only need to open the link to check the screen's accessibility

File: cypress/support/page_objects/ManageYourEremindersPage.js (hunk 4 above)
Comment (danfalc) [reply cypress/support/page_objects/ManageYourEremindersPage.js]: I agree with the environments point, but doesn't seem to like it? 
image

Comment (rsuller) [reply cypress/support/page_objects/ManageYourEremindersPage.js]: Drop the initial `/` and retry

Comment (danfalc) [reply cypress/support/page_objects/ManageYourEremindersPage.js]: image
Same

File: cypress/support/page_objects/ManageYourEremindersPage.js (hunk 3 above)
Comment (rsuller) [reply cypress/support/page_objects/ManageYourEremindersPage.js]: @danfalc just this one to be removed

**Pull request merged**